message_passing/update_with_start/lazy_initialization/README.md

```diff
@@ -0,0 +1,16 @@
+# Update With Start: Lazy init
+
+This sample illustrates the use of update-with-start to send Updates to a Workflow, starting the Workflow if
+it is not running yet. The Workflow represents a Shopping Cart in an e-commerce application, and
+update-with-start is used to add items to the cart, receiving back the updated cart subtotal.
+
+Run the following from this directory:
```

Comment:
```suggestion
To run, first see [README.md](../../README.md) for prerequisites. Then, run the following from this directory:
```

We have tried in the past to have samples refer to the README for prereqs (namely starting a local server), though maybe some samples are not doing that lately. Unsure if we also want to document how to enable multi-op since that will become stale and we may forget to come back and update the instructions.